tests/test_data_api_client.py

```diff
@@ -532,6 +532,29 @@ def test_create_buyer_email_domain(self, data_client, rmock):
         }
 
 
+class TestEmailVaildForAdminMethod(object):
+
+    def test_valid_api_response_is_true(self, data_client, rmock):
+        rmock.get(
+            "http://baseurl/users/valid-admin-email?email_address=kev%40gov.uk",
+            json={"valid": True},
+            status_code=200
+        )
+        result = data_client.email_is_valid_for_admin_user('kev@gov.uk')
+        assert rmock.called
+        assert result is True
+
+    def test_not_valid_api_response_is_false(self, data_client, rmock):
```

Comment:
Could we have more specific test names? The class above has `test_is_email_address_with_valid_buyer_domain_true`, so how about `test_is_email_address_with_valid_admin_domain_true` ?

Reply:
Done 👍